homeassistant/components/zeroconf/__init__.py

```diff
@@ -127,7 +127,7 @@ def info_from_service(service):
         except UnicodeDecodeError:
             _LOGGER.warning("Unicode decode error on %s: %s", key, value)
 
-    address = service.address or service.address6
+    address = service.addresses[0]
```

Comment:
Do we need to check if an address is available? And do we not want to share info about other addresses?

Reply:
For discovery it shouldn't be possible, else it wouldn't be discovered